packages/react-native-worklets/src/WorkletsModule/NativeWorklets.ts

```diff
@@ -16,6 +16,8 @@ class NativeWorklets {
   #workletsModuleProxy: WorkletsModuleProxy;
   #shareableUndefined: ShareableRef<undefined>;
   #shareableNull: ShareableRef<null>;
+  #shareableTrue: ShareableRef<true>;
+  #shareableFalse: ShareableRef<false>;
```

Comment:
Shouldn't it be this way?
```suggestion
  #shareableTrue: ShareableRef<boolean>;
  #shareableFalse: ShareableRef<boolean>;
```